.github/pull_request_template.md

```diff
@@ -1,4 +1,47 @@
-===REMOVE===
-Important! Please follow the new guidelines for naming Pull Requests: https://docs.dasch.swiss/developers/dsp/contribution/#pull-request-guidelines
-===REMOVE===
-resolves DEV-
+<!-- Important! Please follow the new guidelines for naming Pull Requests: https://docs.dasch.swiss/developers/dsp/contribution/#pull-request-guidelines-->
```

Comment:
Do we use this template in the future or was it introduced by accident?

Reply:
I added it a few weeks ago to try it out. If we don't like it we can remove it. I find it at times difficult to review things without knowing the context.